.spell.dict

```diff
@@ -0,0 +1,9 @@
+# This is the repository specific custom dictionary
+#
+# Write the words in alphabetical order, one word per line.
+# If an unknown word is generic put it into the global dictionary
+# in lib/tasks/.spell.dict file.
```

Comment:
IMHO hidden files should be used sparingly. It is usually OK as metadata in a top directory, but this is a regular data file so it should be named without the leading dot.